src/app.js

```diff
@@ -0,0 +1,41 @@
+import './app.scss';
+import { createElement } from './lib/dom';
+import { title } from './components/title';
+import { createNewQuoteButton } from './components/button';
+
+export function app() {
+  const header = createElement('header', {
+    className: 'header'
+  });
+  const main = createElement('main', {
+    className: 'main'
+  });
+  const titleElement = title('Simpsons Quotes');
+  const showButton = createNewQuoteButton();
+
+  const quote = fetchData();
+  const quoteElement = createElement('div', {
+    className: 'quote'
+  });
+
+  quoteElement.appendChild(document.createTextNode(quote.quote));
+  header.appendChild(titleElement);
+  main.appendChild(quoteElement);
+  main.appendChild(showButton);
+
+  return [header, main];
+}
+
+//Api-Funktion, response holt daten
+async function fetchData() {
+  const response = await fetch(
+    'https://thesimpsonsquoteapi.glitch.me/quotes?count=1',
+    //'https://httpbin.org/status/400',
+    { mode: 'cors' }
```

Comment:
Do you need this?

Reply:
With this const i get the quote from the api.

Reply:
I mean `{ mode: 'cors' }`?

Reply:
:) Yes, without the inspector gives me a warning, with `no-cors` the code runs once and then the inspector also gives a warning.

Reply:
Got it, thx. 
Let us talk about that warning on Monday.